src/reference/02-DetailTopics/01-Using-sbt/07-sbt-server.md

```diff
@@ -126,7 +132,7 @@ The `uri` field is the same, and the `token` field contains a 128-bits non-negat
 
 To initiate communication with sbt server, the client (such as a tool like VS Code) must first send an [`initialize` request][lsp_initialize]. This means that the client must send a request with method set to "initialize" and the `InitializeParams` datatype as the `params` field.
 
-To authenticate yourself, you must pass in the token in `initializationOptions` as follows:
+If the server is running in TCP mode, to authenticate yourself, you must pass in the token in `initializationOptions` as follows:
```

Comment:
I have the example written for domain socket in https://developer.lightbend.com/blog/2017-11-30-sbt-1-1-0-RC1-sbt-server/

Reply:
copied the netcat session from the blog as an example.

Reply:
beautiful.